src/schedule/zephyr_domain.c

```diff
@@ -51,124 +51,100 @@ K_THREAD_STACK_DEFINE(ll_workq_stack2, ZEPHYR_LL_WORKQ_SIZE);
 K_THREAD_STACK_DEFINE(ll_workq_stack3, ZEPHYR_LL_WORKQ_SIZE);
 #endif
 
-#define LL_TIMER_PERIOD 1000 /* period in microseconds */
+#define LL_TIMER_PERIOD_US 1000 /* period in microseconds */
```

Comment:
Btw, does the maths simplify in the logic below if we can enforce an integer divider of the 19.2M clk / 48000 (i.e. 400). If it makes it even easier we should just be adding 400 for each time out from the last one.

Reply:
For this we'll just set `CONFIG_SYS_CLOCK_TICKS_PER_SEC` to 48000 in a follow-up PR. I think it would be better to keep this generic though. 19200000 is also divisible by 50000. Don't think using 384 is any more difficult than 400 for the DSP, especially since 400 = 0x190 (3 bits set) whereas 384 = 0x180 (2 bits set) :-D I understand that 48000 is good because it's a popular audio sampling frequency, so we can use it. But I'm not sure we want to embed it into our timer logic. Especially since if we remove `.domain_set()` (see below) all the timer update logic will reside in the "generic" cAVS timer driver.